conda_recipes/cinema4d-openjd/recipe/deactivate.sh

```diff
@@ -0,0 +1,8 @@
+unset DEADLINE_CLOUD_PYTHONPATH
```

Comment:
Outside the scope of this PR, but we might need to choose different env var names here in the future. KeyShot's adapter uses the same one. While it's not a common use case, if both adapters were in the environment at the same thing, they'd stop on each others env vars.

Reply:
Nice catch, this environment variable will be removed soon. I'm waiting [on this PR ](https://github.com/aws-deadline/deadline-cloud-for-cinema-4d/pull/73) to get merged after which we should `C4DPYTHONPATH311` instead.